projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -1,238 +1,134 @@
-  <ng-container *ngIf="displayPolicies?.length > 0; else noPolicies">
-    <div class="container-fluid d-none d-lg-block">
-      <!-- Desktop Layout-->
-      <div class="section-header-heading row">
-        <h5 class="col">{{ 'policy.policy' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.paymentFrequency' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'premiumCalendar.dueDate' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.premium' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.paymentMethod' | cxTranslate }}</h5>
-      </div>
-      <div *ngFor="let policy of displayPolicies">
-        <div
-          class="premium-data-row row border boder-color-3"
-          (click)="openPolicy(policy)"
-          [ngClass]="{
-            clickable: policy.billingData?.length > 1,
-            active: policy.opened
-          }"
-        >
-          <div class="col">
-            <div class="row">
-              <div class="col-4">
-                <div class="image-wrapper">
-                  <img
-                    class="image-item"
-                    [src]="
-                      getBaseUrl() +
-                      policy.mainProduct.coverageProduct.images[0]?.url
-                    "
-                    alt=""
-                  />
+<ng-container *ngIf="policies$ | async as policies; else noPolicies">
+  <ng-container *ngIf="policiesLoaded$ | async; else loading">
+    <ng-container
+      *ngIf="policies?.insurancePolicies?.length > 0; else noPolicies"
+    >
+      <div class="container-fluid">
+        <!-- Desktop Layout-->
+        <div class="section-header-heading row d-none d-lg-flex text-left">
+          <h5 class="col text-center">{{ 'policy.policy' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.paymentFrequency' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'premiumCalendar.dueDate' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.premium' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.paymentMethod' | cxTranslate }}</h5>
+        </div>
+        <div *ngFor="let policy of policies.insurancePolicies; let i = index">
+          <div
+            class="premium-data-row row border boder-color-3"
+            (click)="toggleActiveAccordion(i)"
+            [ngClass]="{
+              clickable: policy.billingData?.length > 1,
+              active: selectedIndexes.includes(i)
+            }"
+          >
+            <div class="col">
+              <div class="row">
+                <div class="col-4">
+                  <div class="image-wrapper small">
+                    <img
+                      class="image-item"
+                      [src]="
+                        getBaseUrl() +
+                        policy.mainProduct.coverageProduct.images[0]?.url
+                      "
+                      alt=""
+                    />
+                  </div>
+                </div>
+                <div class="col-8 mw-100">
```

Comment:
Is there a need for mw-100? Please check, it looks needless.

Reply:
I need that for mobile view. Without that style I have display in the attachment
![Screenshot 2020-12-03 at 13 15 23](https://user-images.githubusercontent.com/75082466/101017049-dec19980-3569-11eb-96b8-f50e4fa29ea4.png)

Reply:
No sure I see the problem, but if you need to have 100% width on mobile the you should use col-12 for that and col-8 for other break points (maybe only col-lg-8 would be enough...)